<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1205</width>
    <height>894</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1151</width>
      <height>841</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Mechanical Scan</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>300</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sampling Rate (samples/s)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>180</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Starting Wavelength (nm)</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="metadataTextEdit">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>510</y>
        <width>231</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="graphChoices2">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>280</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="graphHolder2" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>581</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="graphChoices3">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>550</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>0</y>
        <width>20</width>
        <height>811</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="startWaveSpinBox">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>180</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>1465.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1570.000000000000000</double>
      </property>
      <property name="value">
       <double>1515.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>767</x>
        <y>340</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Inputs (each one &lt;br&gt;on new line)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="endWaveSpinBox">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>220</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>1465.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1570.000000000000000</double>
      </property>
      <property name="value">
       <double>1525.000000000000000</double>
      </property>
     </widget>
     <widget class="QWidget" name="graphHolder1" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>581</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="graphHolder3" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>550</y>
        <width>581</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>220</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ending Wavelength (nm)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="startScanButton">
      <property name="geometry">
       <rect>
        <x>957</x>
        <y>660</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Start Scan</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="fileNameLineEdit">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>470</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>test</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="inputsTextEdit">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>340</y>
        <width>231</width>
        <height>81</height>
       </rect>
      </property>
      <property name="plainText">
       <string>ai0: Gas Cell
ai1: Interferometer
ai2: Reference</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>40</y>
        <width>271</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Mechanical Scan
 Parameters</string>
      </property>
     </widget>
     <widget class="QComboBox" name="graphChoices1">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>10</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>260</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Scan Rate (nm/s)</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="scanRateSpinBox">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>260</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>0.002400000000000</double>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="sampleRateSpinBox">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>300</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
      <property name="maximum">
       <double>50000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>1000.000000000000000</double>
      </property>
      <property name="value">
       <double>1000.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>817</x>
        <y>470</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>File name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>817</x>
        <y>510</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Metadata&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>430</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Save Dir</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mechSaveFolderLineEdit">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>430</y>
        <width>351</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>I:\thompsonlab\REI\Daily\</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Piezo Scan</string>
     </attribute>
     <widget class="QWidget" name="graphHolder4" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>641</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="startPiezoScanButton">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>290</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Piezo Scan</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stopPiezoScanButton">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>330</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop Piezo Scan</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>180</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>File name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="savePiezoDataContButton">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>410</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Save Continuously</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="metadataTextEdit_piezo">
      <property name="geometry">
       <rect>
        <x>853</x>
        <y>220</y>
        <width>231</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>20</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Piezo Scan Parameters</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>220</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Metadata&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="piezoCenterSpinBox">
      <property name="geometry">
       <rect>
        <x>853</x>
        <y>100</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>1465.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1570.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>1515.000000000000000</double>
      </property>
     </widget>
     <widget class="QLineEdit" name="fileNameLineEdit_piezo">
      <property name="geometry">
       <rect>
        <x>853</x>
        <y>180</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>piezoData</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>100</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Center Wavelength (nm)</string>
      </property>
     </widget>
     <widget class="QComboBox" name="graphChoicesPiezo">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>520</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="startPiezoSamplingButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Sampling</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stopPiezoSamplingButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop Sampling</string>
      </property>
     </widget>
     <widget class="QPushButton" name="resetDeviceButton">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>780</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset USB6002</string>
      </property>
     </widget>
     <widget class="QPushButton" name="startPDBButton">
      <property name="geometry">
       <rect>
        <x>1070</x>
        <y>780</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start PDB</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>270</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Autoscale y-axis</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="customRangeMinSpinBox">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>360</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>-0.262000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>320</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Fixed y-axis</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>340</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Min</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>340</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Max</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>140</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Save Dir</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="piezoSaveFolderLineEdit">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>140</y>
        <width>411</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>I:\thompsonlab\REI\Daily\</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="rolloverSpinBox">
      <property name="geometry">
       <rect>
        <x>853</x>
        <y>370</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
      <property name="value">
       <double>500.000000000000000</double>
      </property>
     </widget>
     <widget class="QPushButton" name="savePiezoDataOnceButton">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>410</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Save One File</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>370</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Acquisitions per file</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="customRangeMaxSpinBox">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>360</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>-0.242000000000000</double>
      </property>
     </widget>
     <widget class="QWidget" name="graphHolder5" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>520</y>
        <width>641</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1205</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
